src/components/Logs/Table/logs-table.js

```diff
@@ -55,64 +124,81 @@ export function LogsTable({ logs, actions = [] }) {
 
         <div className="pb-12">
           <div className="flex flex-col">
-            <div className="align-middle inline-block min-w-full">
+            <div className="inline-block min-w-full">
               <table
-                className="min-w-full divide-y divide-gray-300 border-separate"
-                style={{ borderSpacing: "0px" }}
+                className="w-full border rounded-t-6px"
+                {...getTableProps()}
               >
-                <thead className="sticky top-0 bg-white">
-                  <tr>
-                    <th className="border-b border-gray-300" />
-                    <th className="text-left text-xs font-medium text-gray-500 uppercase tracking-wider border-b border-gray-300 pr-5">
-                      Timestamp
-                    </th>
-                    <th className="text-left text-xs font-medium text-gray-500 uppercase tracking-wider border-b border-gray-300">
-                      Message
-                    </th>
-                  </tr>
-                </thead>
-                <tbody
-                  className="bg-white divide-y divide-gray-300"
-                  style={{ fontSize: "0.75rem", lineHeight: "0.9rem" }}
-                >
-                  {logs.map((row, idx) => (
-                    // eslint-disable-next-line react/no-array-index-key
-                    <tr key={idx}>
-                      {actions.length > 0 && (
-                        <td className="border-b border-gray-100 ">
-                          <input
-                            type="checkbox"
-                            className="focus:ring-indigo-400 h-4 w-4 text-indigo-600 border-gray-300 rounded mr-2"
-                            onChange={(e) =>
-                              handleSelect(row, e.target.checked)
-                            }
-                            checked={selectedList.indexOf(row) >= 0}
-                          />
-                        </td>
-                      )}
-                      <td
-                        className="border-b border-gray-100 no-wrap py-1"
-                        style={{ whiteSpace: "nowrap" }}
+                <thead className="bg-lightest-gray border rounded-t-6px">
+                  {headerGroups.map((headerGroup) => (
+                    <tr key={uuidv4()} {...headerGroup.getHeaderGroupProps()}>
```

Comment:
```suggestion
                    <tr key={index} {...headerGroup.getHeaderGroupProps()}>
```
ты знаешь, зачем вообще нужен key элементам? лучше уж индекс тогда и задушить претензии еслинта. но почте в примерах react-table я ничего такого не вижу ?